mlir/test/xmir/e2e/resnet50_blk.e2e.mlir

```diff
@@ -14,10 +15,8 @@ module {
     }
      : (tensor<1x32x32x64xf32>, tensor<64x3x3x64xf32>, tensor<64xf32>) -> tensor<1x32x32x64xf32>
 
-    %1 = "tosa.clamp"(%0) {
-      min_fp = 0.0 : f32,
+    %1 = "tosa.reluN"(%0) {
```

Comment:
I heard reluN is deprecated in the latest TOSA specification because it's fully representable using clamp.

Reply:
I still see it upstream. Maybe it is a discussion.

Reply:
I mean the original TOSA specification. MLIR upstream seems slightly behind it, so we can still have it but need to change later when they update it.
I can confirm it's not present in the latest version 0.30.0 (older versions, e.g. 0.22.0 has ReluN)
https://developer.mlplatform.org/file/download/vdc2wr43p7p3b6nqgwxk/PHID-FILE-andd6yb2cozdie3ihy7h/tosa_spec.pdf